omegaconf/basecontainer.py

```diff
@@ -211,6 +212,12 @@ def convert(val: Node) -> Any:
             assert isinstance(inter, str)
             return inter
         elif conf._is_missing():
+            if throw_on_missing:
+                conf._format_and_raise(
+                    key=None,
+                    value="???",
+                    cause=MissingMandatoryValue("Missing mandatory value: $FULL_KEY"),
+                )
```

Comment:
can you refactor out a small local raise_missing_value_exception() function to reduce the amount of new code?

Reply:
Done, in https://github.com/omry/omegaconf/pull/503/commits/7b49aea8781c97a1515a7f723f78a141054e4dfe.